### PR TITLE
fix(common): styling minors

### PR DESCRIPTION
**Pull request by banklesss**
## Description / Change(s) / Related issue(s)

Styling minors.

##### Ticket

YOMO-1865
YOMO-1455

### CODE REVIEW

File: apps/wallet-mobile/src/components/Button/Button.tsx [hunk 1]
```diff
@@ -153,6 +154,10 @@ const useStyles = () => {
     },
     buttonDisabled: {
       opacity: 0.5,
+      color: color.gray_cmin,
```
Comment (banklesss) [inline apps/wallet-mobile/src/components/Button/Button.tsx]:
```suggestion
```

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/components/Button/Button.tsx [hunk 2]
```diff
@@ -154,6 +155,9 @@ const useStyles = () => {
     buttonDisabled: {
       opacity: 0.5,
     },
+    buttonDisabledText: {
+      color: color.gray_cmin,
```
Comment (stackchain) [inline apps/wallet-mobile/src/components/Button/Button.tsx]: text, should be `text_..min`

Comment (stackchain) [reply apps/wallet-mobile/src/components/Button/Button.tsx]: confirm if is the case for the buttons as well.

Comment (banklesss) [reply apps/wallet-mobile/src/components/Button/Button.tsx]: text_gray_min is not created 👀

will have a look

Comment (banklesss) [reply apps/wallet-mobile/src/components/Button/Button.tsx]: done

**Pull request merged**